tenacity/__init__.py

```diff
@@ -55,6 +58,11 @@
 
 from tenacity import _utils
 
+if sys.version_info > (3, 4):
```

Comment:
asyncio is part of Python stdlib since Python 3.4.

It is possible to install it as a third party module on Python 3.3, but I'm not sure that it's worth it to support Python 3.3 (it's no more used, and it wasn't never very popular).